src/main/java/io/antmedia/enterprise/streamapp/WebSocketLocalHandler.java

```diff
@@ -79,7 +79,17 @@ public void onMessage(Session session, String message) {
 	
 	private void createHandler(ApplicationContext context, Session session) {
 		try {
-			if(io.antmedia.rest.RestServiceBase.isEnterprise()) {
+
+			String rtmpRequest;
+			
+			try {
+				rtmpRequest = session.getRequestParameterMap().get("rtmpRequest").get(0);	
+			} catch (Exception e) {
+				rtmpRequest = "";
+			}
+
+			// If user want to RTMP play, should add rtmp query in websocket URL.
+			if(io.antmedia.rest.RestServiceBase.isEnterprise() && !rtmpRequest.equals("rtmp")) {
```

Comment:
make the name of the parameter `rtmpForward`  and its value should to `true` or `false`